examples/README-authz.md

```diff
@@ -0,0 +1,353 @@
+For example:
+    
+    Dev Team A can write to topics that start with x_ on any cluster
+        
+    Dev Team B can read from topics that start with x_ on any cluster
+    Dev Team B can update consumer group offsets that start with x_ on any cluster
+
+    ClusterManager of cluster2 Group has full access to cluster config on cluster2
+    ClusterManager of cluster2 Group has full access to consumer groups on cluster2
+    ClusterManager of cluster2 Group has full access to topics on cluster2
+    
+If we take a closer look at the `Dev Team A can write ...` permission definition, we see that it combines a resource called `Topic:x_*`, scopes `Describe` and `Write`, and `Dev Team A` policy.
+Let's click on `Dev Team A` policy. We see that it matches all users that have a realm role called `Dev Team A`.
+
+Similarly, the `Dev Team B ...` permissions perform matching using `Dev Team B` policy which also uses realm role to match allowed users - in this case those with realm role `Dev Team B`.
+The `Dev Team B ...` permissions grant users `Describe` and `Read` on `Topic:x_*`, and `Group:x_*` resources, effectively giving matching users and clients the ability to read from topics, and update the consumed offsets for topics and consumer groups that have names starting with 'x_'. 
+
+  
+In Keycloak, confidential clients with 'service accounts' enabled can authenticate to the server in their own name using a clientId and a secret.
+This is convenient for microservices which typically act in their own name, and not as agents of a particular user (like a web site would, for example).
+Service accounts can have roles assigned like regular users.
+They can not, however, have groups assigned.
+As a consequence, if you want to target permissions to microservices using service accounts, you can't use Group policies, but are forced to use Role policies.
+Or, thinking about it another way, if you want to limit certain permissions only to regular user accounts where authentication with username and password is required, you should use Group policies, rather than Role policies.
+That's what we see used in `permissions` that start with 'ClusterManager'.
+Performing cluster management is usually done interactively - in person - using CLI tools. 
+It makes sense to require the user to log-in, before using the resulting access token to authenticate to Kafka Broker.
+In this case the access token represents the specific user, rather than the client application.
+
+Before continuing, there is one setting we need to check.
+Due to [a little bug in Keycloak](https://issues.redhat.com/browse/KEYCLOAK-12640) the realm is at this point misconfigured, and we have to fix the configuration manually.
+Under `Clients` / `kafka` / `Authorization` / `Settings` make sure the `Decision Strategy` is set to `Affirmative`, and NOT to `Unanimous`. Click `Save` after fixing it.
+  
```

Comment:
New subsection: Testing with clients

Reply:
I created several chapter. This one I called 'Authorization in Action Using CLI Clients'